src/components/ui/Button/button.scss

```diff
@@ -68,3 +68,15 @@
     background-color: var(--button-bg-disabled);
   }
 }
+
+.Layer__ButtonTransparentContent {
+  opacity: 0;
+  grid-row: 1 / -1;
+  grid-column: 1 / -1;
+}
+
+.Layer__ButtonSpinnerContainer {
+  grid-column: 1 / -1;
+  grid-row: 1 / -1;
+  place-self: center;
```

Comment:
I think the spinner is not centered vertically. Probably adding padding or static height will improve that:
```
  padding-top: 3px;
  // Or:
  height: 18px;
```

![image](https://github.com/user-attachments/assets/c097c9c1-45a7-480e-9a20-4ec9bcf93180)

![image](https://github.com/user-attachments/assets/5f1d6c5d-c5fb-4a0f-aedd-d2b45f83459c)